### PR TITLE
Add Docker Desktop Edge 2.5.1.0 release notes

### PR DESCRIPTION
**Pull request by mat007**
### Proposed changes

Added Docker Desktop Edge 2.5.1.0 release notes.

### CODE REVIEW

**Review by usha-mandya — DISMISSED** [review]
LGTM. Thanks @mat007

File: docker-for-mac/edge-release-notes.md [hunk 1]
```diff
@@ -11,6 +11,26 @@ This page contains information about Docker Desktop Edge releases. Edge releases
...
+2020-11-18
+
+> [Download](https://desktop.docker.com/mac/edge/49923/Docker.dmg)
+
+### Upgrades
+
+- [Docker Engine 20.10.0-rc1](https://github.com/docker/docker-ce/blob/master/CHANGELOG.md#20100)
+- [Compose CLI v1.0.2](https://github.com/docker/compose-cli/releases/tag/v1.0.2)
+- [Snyk v1.424.4](https://github.com/snyk/snyk/releases/tag/v1.424.4)
+- [Kubernetes 1.19.3](https://github.com/kubernetes/kubernetes/releases/tag/v1.19.3)
```
Comment (stephen-turner) [inline docker-for-mac/edge-release-notes.md:24]: Do we need the warning about "your cluster will be reset"?

File: docker-for-mac/edge-release-notes.md [hunk 2]
```diff
@@ -11,6 +11,26 @@ This page contains information about Docker Desktop Edge releases. Edge releases
 For information about Stable releases, see the [Stable release notes](release-notes.md). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.5.1.0
+2020-11-18
+
+> [Download](https://desktop.docker.com/mac/edge/49923/Docker.dmg)
```
Comment (StefanScherer) [inline docker-for-mac/edge-release-notes.md:17]: We normally hide the latest download links, but update the link of the previous release to show the direct link there.

File: docker-for-mac/edge-release-notes.md [hunk 3]
```diff
@@ -11,6 +11,26 @@ This page contains information about Docker Desktop Edge releases. Edge releases
...
+### Upgrades
+
+- [Docker Engine 20.10.0-rc1](https://github.com/docker/docker-ce/blob/master/CHANGELOG.md#20100)
+- [Compose CLI v1.0.2](https://github.com/docker/compose-cli/releases/tag/v1.0.2)
+- [Snyk v1.424.4](https://github.com/snyk/snyk/releases/tag/v1.424.4)
+- [Kubernetes 1.19.3](https://github.com/kubernetes/kubernetes/releases/tag/v1.19.3)
+
+### Bug fixes and minor changes
+
+- Fixed an issue that caused Docker Desktop to crash on MacOS 11.0 (Big Sur) when VirtualBox was also installed. See [docker/for-mac#4997](https://github.com/docker/for-mac/issues/4997).
```
Comment (stephen-turner) [inline docker-for-mac/edge-release-notes.md:28]: Add: Renamed ‘Run Diagnostics’ to ‘Get support’.

**Review by stephen-turner — CHANGES_REQUESTED** [review]
I've made a couple of comments of other things we could add. I've only mentioned them in the Mac release notes, but they both apply to Windows too.

File: docker-for-mac/edge-release-notes.md (hunk 2 above)
Comment (mat007) [reply docker-for-mac/edge-release-notes.md:17]: Didn’t we stop doing that? The previous release had a direct link already. @usha-mandya what do you think?

File: docker-for-mac/edge-release-notes.md (hunk 1 above)
Comment (mat007) [reply docker-for-mac/edge-release-notes.md:24]: Good catch!

Comment (mat007) [reply docker-for-mac/edge-release-notes.md:24]: Actually it’s a warning we put in the changelogs, I don’t see any in the release notes here in the docs…

File: docker-for-mac/edge-release-notes.md (hunk 3 above)
Comment (mat007) [reply docker-for-mac/edge-release-notes.md:28]: OK.

File: docker-for-mac/edge-release-notes.md (hunk 2 above)
Comment (StefanScherer) [reply docker-for-mac/edge-release-notes.md:17]: Ok, fair enough. 👍

File: docker-for-mac/edge-release-notes.md (hunk 1 above)
Comment (mat007) [reply docker-for-mac/edge-release-notes.md:24]: Ah yes right, I see it now in the 2.2.2.0 section. We missed it in the more recent ones.
I’ll add it here!

File: docker-for-mac/edge-release-notes.md (hunk 2 above)
Comment (usha-mandya) [reply docker-for-mac/edge-release-notes.md:17]: I'm OK with the direct link. I believe we started adding a direct link after a couple of instances where we forgot to update the previous URL.

**Review by usha-mandya — APPROVED** [review]

**Comment by mat007** [comment]
@stephen-turner could you take another look, please?

**Review by stephen-turner — APPROVED** [review]

**Pull request merged**